boards/launchxl/Makefile-app

```diff
@@ -0,0 +1,25 @@
+# Makefile for loading applications for the launchxl
+
+$(call check_defined, TOCK_USERLAND_BASE_DIR)
+$(call check_defined, BUILDDIR)
+$(call check_defined, PACKAGE_NAME)
+$(call check_defined, UNIFLASH_CLI_BASE)
```

Comment:
I think remove this check here as UNIFLASH check is also in the flash build recipe and you don't strictly need uniflash to be able to build apps

Reply:
Thanks, missed to remove the check in the app makefile!